Modules/Scripted/DICOMLib/DICOMUtils.py

```diff
@@ -911,11 +913,25 @@ def onNodeAdded(caller, event, calldata):
     return loadedNodeIDs
 
 
+# ------------------------------------------------------------------------------
+GLOBAL_DICOMWEB_USER_KEY = "DICOMWEBUSER"
+GLOBAL_DICOMWEB_PASSWORD_KEY = "DICOMWEBPWD"
+
+
+def getGlobalDICOMAuth() -> Optional[requests.auth.HTTPBasicAuth]:
+    """Get the global authentication settings for DICOM networking, if initialized."""
+    user = slicer.util.settingsValue(GLOBAL_DICOMWEB_USER_KEY, "")
+    pwd = slicer.util.settingsValue(GLOBAL_DICOMWEB_PASSWORD_KEY, "")
+    return requests.auth.HTTPBasicAuth(user, pwd) if user or pwd else None
+
+
+# ------------------------------------------------------------------------------
```

Comment:
```suggestion
```

Reply:
Updated. Note that other functions in this file are broken up with this divider though.